cmd/livepeer_cli/wizard.go

```diff
@@ -41,6 +42,23 @@ func (w *wizard) readString() string {
 	}
 }
 
+func (w *wizard) readMultilineString() string {
+	ss := make([]string, 1)
+	fmt.Printf("(press enter, then ctrl+d when done) > ")
+	for {
+		text, err := w.in.ReadString('\n')
+		if err != nil {
+			if err != io.EOF {
+				log.Crit("Failed to read user input", "err", err)
+				continue
+			}
+			break
+		}
+		ss = append(ss, text)
+	}
+	return strings.TrimSpace(strings.Join(ss, ""))
```

Comment:
Some messages end in whitespace (either horizontal or vertical i.e. `\n`) - we shouldn't mutate the provided message.